proposals/csharp-11.0/low-level-struct-improvements.md

```diff
@@ -156,9 +156,9 @@ Next the rules for ref reassignment need to be adjusted for the presence of `ref
 
 The left operand of the `= ref` operator must be an expression that binds to a ref local variable, a ref parameter (other than `this`), an out parameter, **or a ref field**.
 
-> For a ref reassignment in the form ...
-> 1. `x.e1 = ref e2`: where `x` is *safe-to-escape* at least *return only* then `e2` must have *ref-safe-to-escape* at least as large as `x`
-> 2. `e1 = ref e2`: where `e1` is a `ref` local or `ref` parameter then `e2` must have a *safe-to-escape* equal to *safe-to-escape* for `e1` and `e2` must have *ref-safe-to-escape* at least as large as *ref-safe-to-escape* of the *ref-safe-to-escape* of `e1`
+> For a ref reassignment in the form `e1 = ref e2` both of the following must be true:
+> 1. e2 must have ref-safe-to-escape at least as large as the ref-safe-to-escape of e1
+> 2. e1 must have the same safe-to-escape as e2
```

Comment:
nit: backticks on `e1` and `e2`
nit: I'm okay to keep a side-note on `x.e1` case, since I agree it is a bit tricky how it falls out.

Reply:
Yeah formatting didn't copy correctly. Will add the sample.